.ci/install_cri-containerd.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# 
+# Copyright (c) 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+cidir=$(dirname "$0")
+source "${cidir}/lib.sh"
+get_cc_versions
+# Fixme add a version to use in versions.txt file
```

Comment:
Is there an issue for this, or are you planning to update this PR to remove the "Fixme"?